src/ssl/support.cc

```diff
@@ -926,6 +926,11 @@ Ssl::verifySslCertificate(Security::ContextPointer &ctx, CertificateProperties c
 bool
 Ssl::setClientSNI(SSL *ssl, const char *fqdn)
 {
+    URL tmpUrl;
+    tmpUrl.host(fqdn);
+    if (tmpUrl.hostIsNumeric())  // raw IP not appropriate for SNI
+        return true;
```

Comment:
Please adjust Ssl::setClientSNI() return type from `bool` to `void`:

* Nobody is checking the return value right now.
* It is much simpler to not specify the return type than to make sure that a `true` return value for an essentially invalid `fqdn` parameter is what all future callers would want/expect.
* If errors become important to future callers, then the function should probably throw on errors instead (TODO).

Do not forget to update the method description as well.